arch/riscv/Kconfig

```diff
@@ -118,6 +118,62 @@ config GEN_IRQ_VECTOR_TABLE
 config NUM_IRQS
 	int
 
+config HAS_MONITOR_HART
+	bool "Does SOC have monitor hart with different capabilities to application harts"
+	default n
```

Comment:
The default is `n` anyway, so there is no need for `default n`. Please remove and all others.

Reply:
@dcpleung Thanks for taking the time to review. Understood, I will fix this